src/main/asciidoc/java/index.adoc

```diff
@@ -289,6 +289,66 @@ An alternative way to run blocking code is to use a <<worker_verticles, worker v
 
 A worker verticle is always executed with a thread from the worker pool.
 
+== Async coordination
+
+Coordination of multiple asynchronous results can be achieved with Vert.x `link:../../apidocs/io/vertx/core/Future.html[futures]`.
+
+`link:../../apidocs/io/vertx/core/CompositeFuture.html#all-io.vertx.core.Future-io.vertx.core.Future-[CompositeFuture.all]` takes several futures arguments (up to 6) and returns a future that is
+_succeeded_ when all the futures are and _failed_ otherwise:
+
+[source,java]
+----
+Future<HttpServer> httpServerFuture = Future.future();
+httpServer.listen(httpServerFuture.completer());
+
+Future<NetServer> netServerFuture = Future.future();
+netServer.listen(netServerFuture.completer());
+
+CompositeFuture.all(httpServerFuture, netServerFuture).setHandler(ar -> {
+  if (ar.succeeded()) {
+    // All server started
+  } else {
+    // At least one server failed
+  }
+});
+----
+
+The handler returned by `link:../../apidocs/io/vertx/core/Future.html#completer--[completer]` completes the future.
+
+`link:../../apidocs/io/vertx/core/CompositeFuture.html#any-io.vertx.core.Future-io.vertx.core.Future-[CompositeFuture.any]` takes several futures arguments (up to 6) and returns a future that is
+_failed_ when all the futures are and _succeeded_ otherwise:
```

Comment:
Minor, I wonder if this would be more clear: "and returns a future that is _succeeded_ if one of the futures is, and _failed_ otherwise"